index.html

```diff
@@ -0,0 +1,163 @@
+
+              <div class="collapse navbar-collapse justify-content-end ms-2" id="CC-conference-btn-container" >
+                <button type="button" class="text-center border border-4 border-danger p-1 ps-2 pe-2 text-danger ">CC Campaign</button>
+              </div>
+            </ul>
+            
+        </div>
+
+        
+      </div>
+    </nav>
+
+  </header>
+    
+  <main>
+    <section id="headline" class="w-100 position-relative pb-3">
+      <img id="img1" width="100%" height="500px" src="./imgs/home.jpg" alt="home image">
+      <img id="img2" width="100%" height="500px" src="./imgs/home2.jpg" alt="home image">
+
+      <div id="home-content" class="position-absolute top-0 h-100 d-flex flex-column justify-content-center">
+        <h3 id="home-content-h3" class="text-white">
+          EU Largest Tech Conference:
+        </h3>
+        <h1 id="conference-title" class="text-white">
+          Advanced Battery and Electric vehicle </h1>
+        <p class="text-white">Delivering insight into key business and technical challenges.</p>
+        <h4 class="text-white">
+          28th & 29th June 2023<br>
+          <small class="text-white">Hall 3, NEC, Birmingham, UK</small>
+        </h4>
+      </div>
+
+    </section>
+    
+    <section id="main-program" 
+    class="w-100 text-center">
+      <h2 class="text-white">Main Progarm</h2>
+      <div class="line"></div>
+            
+      <div id="main-container" class="row g-2 row-cols-1 row-cols-md-2 row-cols-lg-3 row-cols-xl-5"></div>
+
+      <a id="anchor" class="" href="">See the whole progarm</a>
+      
+      <div class="w-100 ps-4 pe-4">
+        <button type="button" class="w-100 text-center p-3 bg-danger text-white">Join The Conference</button>
+      </div>
+
+    </section>
+
+    <section id="featured-container" 
+    class="w-100">
+      <h2>Featured Speakers</h2>
+      <div class="line"></div>
+
+      <div id="featured-speakers"
+      class="row g-2 row-cols-1 row-cols-md-2"
+      ></div>
+
+    </section>
+
+    <section id="social" class="w-100 bg-dark">
+      <h2>Partner</h2>
+      <div class="line"></div>
+      
+      <div class="d-flex flex-row flex-wrap justify-content-evenly">
+        <a id="soc-a" href="">
+          <span><img src="./imgs/socials/mozilla.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img  src="./imgs/socials/googlekorea.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img src="./imgs/socials/naver.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img src="./imgs/socials/daumkakaologo.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img src="./imgs/socials/airbnb.svg" alt=""></span></a>
+      </div>
+
+    </section>
+
+    <section id="contact" class="w-100 text-center">
+      <a id="con-a" href="#">
+        <span> <img src="./imgs/BatteryTech.png" alt="contact">
+        </span>
+        </a> 
+    </section>
```

Comment:
- [ ] Please be sure to remove the "Partners" and "footer" sections from the home page on the mobile version to match the given design.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/4ab76620-71bf-46a3-9e19-45e200eacdea) | ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/f340bad5-0c77-4903-9aed-968545582199) |